### ISSUE TITLE
C# XML documentation needs XML escaping

### ISSUE THREAD

**Issue by jskeet** [googleapis/toolkit#869]
While experimenting with some new APIs, I've found that if the proto documentation uses angle brackets, we can end up with XML documentation like this:

```csharp
/// <param name="name">
/// Required. The name of the book to be deleted. Values are of the form
/// `books/<book>`
/// </param>

The `<book>` confuses the C# compiler, and it thinks the `<param>` is unclosed.

**Comment by jskeet** [comment]
This affects the Spanner API, which uses this format extensively.

**Comment by chrisdunelm** [comment]
Fixed with #1045.
